#! /usr/bin/env python
PACKAGE='audibot_twist_controller'

from dynamic_reconfigure.msg import SensorLevels
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       Name                  Type      Level   Description             Default   Min      Max
gen.add("max_throttle",       double_t, 0,      "Max throttle pct.",    1.0,      0.01,    1.0)
gen.add("max_brake_force",    double_t, 0,      "Max brake force",      5000.0,   100.0,   5000.0)
gen.add("throttle_p",         double_t, 0,      "Throttle gain",        0.4,      0.0,     0.5)
gen.add("throttle_i",         double_t, 0,      "Throttle gain",        0.1,      0.0,     0.5)
gen.add("brake_gain",         double_t, 0,      "Brake gain",           3000.0,   0.0,     5000.0)

exit(gen.generate(PACKAGE, PACKAGE, "AudibotSpeed"))
